Samples/JExtractJNISampleApp/src/test/java/com/example/swift/MySwiftClassTest.java

```diff
@@ -35,4 +35,30 @@ void init_withParameters() {
             assertNotNull(c);
         }
     }
+
+    @Test
+    void sum() {
+        try (var arena = new ConfinedSwiftMemorySession(Thread.currentThread())) {
```

Comment:
Can you please add a constructor that takes no arguments and uses the current thread? There's no good reason to keep propagating this shape of end user passing in the thread like we do here